### PR TITLE
fix bug Unable to modify notification settings in console

### PR DESCRIPTION
**Pull request by wanjunlei**
Signed-off-by: wanjunlei <wanjunlei@yunify.com>

fix bug kubesphere/kubesphere#3555

### CODE REVIEW

File: roles/ks-monitor/files/prometheus/alertmanager/alertmanager-secret.yaml [hunk 1]
```diff
@@ -64,4 +57,7 @@ stringData:
         "match":
           "alerttype": "auditing"
         "receiver": "auditing"
+      - "match_re":
+          "alerttype": ".*"
```
Comment (benjaminhuo) [inline roles/ks-monitor/files/prometheus/alertmanager/alertmanager-secret.yaml:61]: `.*` match all alerts?

Comment (wanjunlei) [reply roles/ks-monitor/files/prometheus/alertmanager/alertmanager-secret.yaml:61]: Alertmanager uses Go's regular expression, it is very difficult to achieve unequal matching, so this is a temporary alternative.

image

The continue is set to false, so the alert will not repeat.

**Review by benjaminhuo — APPROVED** [review]
/lgtm

File: roles/ks-monitor/files/prometheus/alertmanager/alertmanager-secret.yaml (hunk 1 above)
Comment (benjaminhuo) [reply roles/ks-monitor/files/prometheus/alertmanager/alertmanager-secret.yaml:61]: ok

**Pull request merged**